cluster/saltbase/salt/kube-node-unpacker/kube-node-unpacker.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
```

Comment:
Ya it's copy paste :(. This is #8686 but it's a significant amount of work to create a cross platform solution.